internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/BlobPeersDiscoveryResponse.java

```diff
@@ -31,6 +32,9 @@ public void setDiscoveryResult(List<String> hostNames) {
   }
 
   public List<String> getDiscoveryResult() {
+    if (this.hostNameList != null && !this.hostNameList.isEmpty()) {
+      Collections.shuffle(this.hostNameList);
```

Comment:
Every call to `getDiscoveryResult` would return a different list. The method name doesn't imply this. How about changing the method name so the caller would know?

Reply:
I thought about that too.. But that'd mean we are adding the duplicated logic (tho very minimal) to server finder and dvc finder. That said, it's minor and i'm okay with that

Reply:
Moved it to peer finder logic (server finder and dvc finder).